lms/static/sass/shared/_header.scss

```diff
@@ -143,14 +142,19 @@ header.global {
       }
     }
 
-    a.user-link {
-      @include padding(5px, 2px, 10px, 10px);
+    .user-link {
+      @include padding(0, 0, 0, 0);
```

Comment:
This can just be `padding: 0`

Not sure if we want to keep the `@include` since left and right padding are the same